src/status_im/contexts/wallet/create_account/events.cljs

```diff
@@ -49,3 +51,22 @@
   {:db (update-in db [:wallet :ui :create-account] dissoc :new-keypair)})
 
 (rf/reg-event-fx :wallet/clear-new-keypair clear-new-keypair)
+
+(defn get-derived-addresses
+  [{:keys [db]} [{:keys [password derived-from paths]}]]
+  {:db            (assoc-in db [:wallet :ui :create-account :derivation-path-state] :scanning)
+   :json-rpc/call [{:method     "wallet_getDerivedAddresses"
+                    :params     [password derived-from paths]
+                    :on-success [:wallet/get-derived-addresses-success]}]})
+
+(rf/reg-event-fx :wallet/get-derived-addresses get-derived-addresses)
+
+(defn get-derived-addresses-success
+  [{:keys [db]} [response]]
+  {:db (-> db
+           (assoc-in [:wallet :ui :create-account :derivation-path-state]
+                     (if (:has-activity (first response)) :has-activity :no-activity))
+           (assoc-in [:wallet :ui :create-account :derivation-path]
+                     (cske/transform-keys csk/->kebab-case-keyword (first response))))})
```

Comment:
Maybe we can extract `(first response)` to a `let` and give a name.

Also please use `transfroms/->kebab-case-keyword` since it's a memoized version and performs a lot better.